modules/querying/pages/data-modification-statements.adoc

```diff
@@ -382,7 +374,14 @@ CREATE QUERY who_works_for_company(STRING comp) FOR GRAPH Works_Net {
 }
 ----
 
+To insert an edge with a discriminator value specified, use the `DISCRIMINATOR` keyword to specify the discriminator attributes and specify the values in order.
+For example, the following statement inserts an edge between two vertices with the discriminator values being 2019 and 01, respectively.
 
+[.wrap,console]
+----
+INSERT INTO Study_At (FROM, TO, DISCRIMINATOR(class_year, class_month))
```

Comment:
remove this for now, there is some problem with the insert into statement for multiedges, I will let you know once it is supported.